scripts/data_prep/README.md

```diff
@@ -12,21 +12,48 @@ Using the `convert_dataset_hf.py` script...
 python convert_dataset_hf.py \
   --dataset c4 --data_subset en \
   --out_root my-copy-c4 --splits train_small val_small \
-  --concat_tokens 2048 --tokenizer EleutherAI/gpt-neox-20b --eos_text '<|endoftext|>'
+  --concat_tokens 2048 --tokenizer EleutherAI/gpt-neox-20b --eos_text '<|endoftext|>' \
+  --compression zstd
 ```
 
 Using the `convert_dataset_json.py` script...
 
 <!--pytest.mark.skip-->
 ```bash
 # Convert C4 dataset to StreamingDataset format
```

Comment:
comment should be updated?

Reply:
yes ... not compressing is a big mistake ... lots of wasted space and engress

Reply:
sorry, I meant the `# Convert C4 dataset to StreamingDataset format` comment doesn't seem to be correct for this command